src/core/api/scan_all.go

```diff
@@ -0,0 +1,56 @@
+package api
+
+import (
+	"net/http"
+	"strconv"
+
+	common_job "github.com/goharbor/harbor/src/common/job"
+	"github.com/goharbor/harbor/src/common/utils/log"
+	"github.com/goharbor/harbor/src/core/api/models"
+	"github.com/goharbor/harbor/src/core/config"
+)
+
+// ScanAllAPI handles request of harbor admin...
+type ScanAllAPI struct {
```

Comment:
Although in the UI there's no requirement to list scan all execution history, we probably can add it in the API in this effort.